localstack/services/cloudformation/provider.py

```diff
@@ -153,6 +159,37 @@ def create_stack(self, context: RequestContext, request: CreateStackInput) -> Cr
                 )
             state.stacks.pop(existing.stack_id)
 
+        if (
+            "CAPABILITY_AUTO_EXPAND" not in request.get("Capabilities", [])
+            and "Transform" in template
+        ):
+            raise InsufficientCapabilitiesException(
+                "Requires capabilities : [CAPABILITY_AUTO_EXPAND]"
+            )
+
+        try:
+            parameters = template_preparer.resolve_parameters(
+                template.get("Parameters", {}), request.get("Parameters", [])
+            )
+            template = template_preparer.transform_template(
+                template,
+                parameters,
+            )
+
+        except FailedTransformation as e:
+            stack = Stack(request, template)
+            stack.set_stack_status("ROLLBACK_COMPLETE")
+            stack.add_stack_event(
+                stack.stack_name,
+                stack.stack_id,
+                status="ROLLBACK_IN_PROGRESS",
+                status_reason=e.message,
+            )
```

Comment:
Hm not sure about this.

Basically the stack has a complete status while there's not even yet the rollback_in_progress event. I'd suggest at least flipping these two statements around.